ibis/omniscidb/client.py

```diff
@@ -633,16 +658,44 @@ def _get_table_schema(self, table_name, database=None):
             database, table_name = table_name_
         return self.get_schema(table_name, database)
 
-    def _execute(self, query, results=True):
-        """Execute a query.
+    def _execute_query(self, dml, **kwargs):
+        query = self.query_class(self, dml, **kwargs)
+        return query.execute(**kwargs)
 
-        Paramters
-        ---------
-        query : DDL or DML or string
+    def _execute(
+        self,
+        query: str,
+        results: bool = True,
+        ipc: bool = None,
```

Comment:
Optional[bool]

Reply:
does mypy not complain here? that is very odd

Reply:
maybe we could in a near future add `mypy` to pre-commit hook to check that

Reply:
it needs the flag `--no-implicit-optional`